src/sockets/tests/chatSpamFilter.test.ts

```diff
@@ -0,0 +1,42 @@
+import { ChatSpamFilter } from '../chatSpamFilter';
+
+describe('Chat Spam Filter', () => {
+  it('should allow a user to chat when no messages have been sent', () => {
+    const chatSpamFilter = new ChatSpamFilter();
+    expect(chatSpamFilter.chatRequest('pronub')).toEqual(true);
+  });
+
+  it('Calling tick should increment seconds correctly', () => {
```

Comment:
Let's remove this test case. It tests too far into the internals of the class (i.e. it tests the internal implementation instead of overall behaviour).